walletlibrary/src/main/java/com/microsoft/walletlibrary/wrapper/ManifestResolver.kt

```diff
@@ -31,25 +31,16 @@ internal object ManifestResolver {
                 RawManifest(request)
             }
             is Result.Failure -> {
-                val issuanceCompletionResponse = requestState?.let {
-                    IssuanceCompletionResponse(
+                requestState?.let {
+                    val issuanceCompletionResponse = IssuanceCompletionResponse(
                         IssuanceCompletionResponse.IssuanceCompletionCode.ISSUANCE_FAILED,
                         it,
                         IssuanceCompletionResponse.IssuanceCompletionErrorDetails.FETCH_CONTRACT_ERROR
                     )
+
+                    VerifiedIdRequester.sendIssuanceCallback(issuanceCompletionResponse, issuanceCallbackUrl)
                 }
-                try {
-                    if (issuanceCompletionResponse != null && issuanceCallbackUrl != null)
-                        VerifiedIdCompletionCallBack.sendIssuanceCompletionResponse(
-                            issuanceCompletionResponse,
-                            issuanceCallbackUrl
-                        )
-                } catch (exception: WalletLibraryException) {
```

Comment:
We would need this try/catch block since callback shouldn't block the flow. Just log the error and proceed.